src/containers/AccountManagement.jsx

```diff
@@ -53,10 +54,30 @@ export default class AccountManagement extends Component {
       })
   }
 
+  validInfos (input) {
+    const { t } = this.context
+    // Check if field is empty or not
+    if (input.target.value !== '') {
```

Comment:
@gregorylegarec @CPatchane Well, as it is planned to have an external lib for validation, I'm guessing that it's not really worth it yet to go deeper. Don't you think?